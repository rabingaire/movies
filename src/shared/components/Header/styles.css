.header_wrapper {
  display: flex;
  padding: 20px;
  align-items: center;
  justify-content: center;
  background-color: #364e68;
}
.header_wrapper a {
  font-size: 2rem;
  color: #ffffff;
  padding: 0px 20px;
  text-decoration: none;
  font-family: "Ranga", cursive;
}
