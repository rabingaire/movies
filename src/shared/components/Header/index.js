import React from "react";
import { Link } from "@reach/router";

import "./styles.css";

const NavLink = props => (
  <Link
    {...props}
    getProps={({ isCurrent }) => {
      return {
        style: {
          color: isCurrent ? "#98ccd3" : "#ffffff"
        }
      };
    }}
  />
);

const Header = () => (
  <div className="header_wrapper">
    <NavLink to="/">Home</NavLink>
    <span>&nbsp;</span>
    <NavLink to="trending">Trending</NavLink>
  </div>
);

export default Header;
