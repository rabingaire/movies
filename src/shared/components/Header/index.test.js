// Link.react.test.js
import React from "react";
import Header from "./index";
import renderer from "react-test-renderer";

test("Link changes the class when hovered", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
