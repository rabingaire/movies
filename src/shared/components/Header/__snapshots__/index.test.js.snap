// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="header_wrapper"
>
  <a
    aria-current="page"
    href="/"
    onClick={[Function]}
    style={
      Object {
        "color": "#98ccd3",
      }
    }
  >
    Home
  </a>
  <span>
    Â 
  </span>
  <a
    href="/trending"
    onClick={[Function]}
    style={
      Object {
        "color": "#ffffff",
      }
    }
  >
    Trending
  </a>
</div>
`;
